static/styles/settings.css

```diff
@@ -621,14 +617,54 @@ input[type="checkbox"] {
     margin: 10px 0;
 }
 
-.add-new-emoji-box,
-.add-new-default-stream-box {
+.add-new-emoji-box {
     input[type="text"] {
         padding: 6px;
     }
     margin-bottom: 15px;
 }
 
+#add-default-stream-modal {
+    form,
+    .input-group {
+        margin: 0;
```

Comment:
This is not required since you have already set the element as `inline` and margin does not apply to inline elements (which means the margin of `20px` of input-group class in general is also not applied.

Reply:
Fixed. Thanks.